### PR TITLE
COR-749 Fix responsive Choropleth sizing

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard24**

### CODE REVIEW

File: packages/app/src/components/choropleth/components/canvas-choropleth-map.tsx [hunk 1]
```diff
@@ -46,6 +46,7 @@ export type CanvasChoroplethMapProps = {
   fitExtent: FitExtent;
   getFeatureName: (code: string) => string;
   height: number;
+  useMinHeight?: boolean;
```
Comment (VWSCoronaDashboard19) [inline packages/app/src/components/choropleth/components/canvas-choropleth-map.tsx:49]: Better option for prefixing booleans with: is, has or can.

Suggestion: `hasMinHeight`?

File: packages/app/src/components/choropleth/components/choropleth-map.tsx [hunk 2]
```diff
@@ -136,6 +136,7 @@ export const ChoroplethMap: <T extends ChoroplethDataItem>(
             dataOptions={dataOptions}
             width={width}
             height={correctedHeight}
+            useMinHeight={minHeight > 0}
```
Comment (VWSCoronaDashboard25) [inline packages/app/src/components/choropleth/components/choropleth-map.tsx:139]: Why should we add this logic here?
I think we should carry the the boolean `useMinHeight` through to the surface (full control on the page where we want the Choropleth).

File: packages/app/src/components/choropleth/components/canvas-choropleth-map.tsx (hunk 1 above)
Comment (VWSCoronaDashboard24) [reply packages/app/src/components/choropleth/components/canvas-choropleth-map.tsx]: Fixed!

File: packages/app/src/components/choropleth/components/choropleth-map.tsx (hunk 2 above)
Comment (VWSCoronaDashboard24) [reply packages/app/src/components/choropleth/components/choropleth-map.tsx]: Changed it to something better (I hope ;-) )

**Review by VWSCoronaDashboard19 — APPROVED** [review]

**Pull request merged**